zstacklib/zstacklib/storage/iscsi/session.py

```
@@ -0,0 +1,472 @@
+        
+    Returns:
+        IscsiSession if found, None otherwise
+        
+    Example:
+        >>> session = get_session('192.168.1.100', 'iqn.2020-01.com.example:storage')
+        >>> if session:
+        ...     print(f"Connected with session {session.session_id}")
+    """
+    # Use grep to find specific session
+    cmd = "iscsiadm -m session | grep {}:{} | grep {}".format(ip, port, iqn)
+    r, o, e = bash.bash_roe(cmd)
```

Ruff (0.15.2)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)